packages/blocks/src/api/raw-handling/test/utils.js

```diff
@@ -1,8 +1,31 @@
+
+/**
+ * External dependencies
+ */
+import deepFreeze from 'deep-freeze';
+
 /**
  * Internal dependencies
  */
 import { getPhrasingContentSchema } from '../phrasing-content';
-import { isEmpty, isPlain, removeInvalidHTML } from '../utils';
+import { getBlockContentSchema, isEmpty, isPlain, removeInvalidHTML } from '../utils';
+
+jest.mock( '@wordpress/data', () => {
```

Comment:
We are getting very close to the point where we should introduce our own util for testing code that uses data layer.